development/fake-cht/docker-compose.fake-cht.yml

```diff
@@ -6,14 +6,14 @@ volumes:
 services:
   fake-cht:
     image: node:latest
-    user: "node"
+    # user: "node"
```

Comment:
https://forum.communityhealthtoolkit.org/t/fake-cht-permission-errors-when-running-watchdog-in-dev-mode/2908